### ISSUE TITLE
overflow indicator disappears on bookmarks toolbar

### ISSUE THREAD

**Issue by robcee** [robcee/bookmarks-deiconizer#5]
no longer any indication when user has exceeded number of visible bookmarks, on mac at least.

**Comment by maartenhunink** [comment]
Yes, it does show the Bookmarks dropdown, but not a possibility to see only the items that are not visible in the bar itself.

**Comment by moll** [comment]
Do you have any idea how to fix this, robcee?

**Comment by robcee** [comment]
I do, I just lack the time to do it. I've had enough complaints about this that it's the top priority for this addon.

What I think needs to be done is rather than change the attribute on the toolbar itself from text to icons, I need to iterate each item in the toolbar and change their class to the appropriate text only button type. That's going to mean spending some time in the DOM Inspector and rewriting the bit of the add-on that does the deiconization.

pull requests welcome!

**Comment by moll** [comment]
Any update? Ping. :-)

And just out of curiosity, is the same end achievable with a simple CSS rule? Or why'd you take the JS heavy road?
